examples/platform_services/ios/Runner/AppDelegate.m

```diff
@@ -6,11 +6,34 @@
 
 #import <Flutter/Flutter.h>
 
-@implementation AppDelegate {
-}
-
+@implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    return YES;
+  FlutterDartProject* project = [[FlutterDartProject alloc] initFromDefaultSourceForConfiguration];
+  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+  FlutterViewController* controller = [[FlutterViewController alloc] initWithProject:project
+                                                                             nibName:nil
+                                                                              bundle:nil];
+  FlutterMethodChannel* batteryChannel =
+    [FlutterMethodChannel withController:controller
+                                    name:@"battery"
+                                   codec:[FlutterStandardMethodCodec shared]];
```

Comment:
I think calling this static method sharedInstance would be more idiomatic in Objective-C